test/unit/full_text_search/issue_any_searchable_filter_test.rb

```diff
@@ -0,0 +1,36 @@
+require File.expand_path("../../../test_helper", __FILE__)
+
+module FullTextSearch
+  class SearcherTest < ActiveSupport::TestCase
+    setup do
+      if Gem::Version.new(Redmine::VERSION) < Gem::Version.new("5.1")
+        skip("Need Redmine 5.1 or later")
+      end
+    end
+
+    def test_filter_any_searchable
+      Issue.destroy_all
+      issue_with_searched_word_in_subject = Issue.generate!(subject: "Groonga")
+      issue_with_searched_word_in_description =
+        Issue.generate!(description: "Groonga")
+      issue_without_searched_word =
+        Issue.generate!(subject: "no-keyword",
+                        description: "no-keyword")
+      query = IssueQuery.new(
+        :name => "_",
+        :filters => {
+          "any_searchable" => {
+            :operator => "~",
+            :values => ["Groonga"]
+          }
+        }
+      )
+      searched_issues = Issue.where(query.statement).order(:id)
```

Comment:
It seems that issues page use `IssueQuery#issues` not `Issue.where(query.statement)`: https://github.com/redmine/redmine/blob/09167d0b3238a4a7e27fded6a4111fe39f1bf22f/app/controllers/issues_controller.rb#L62

Should we use the same API?

Reply:
fix: [08ef3cc](https://github.com/clear-code/redmine_full_text_search/pull/164/commits/08ef3ccb0fba1c0986e74ca54788d8bc21e20af0) I think using the same API is better. I fixed it.